src/map/script.c

```diff
@@ -13285,6 +13301,8 @@ static BUILDIN(removemapflag)
 			case MF_NOCASHSHOP:         map->list[m].flag.nocashshop = 0; break;
 			case MF_NOAUTOLOOT:         map->list[m].flag.noautoloot = 0; break;
 			case MF_NOVIEWID:           map->list[m].flag.noviewid = EQP_NONE; break;
+			case MF_NOSTORAGE: 			map->list[m].flag.nostorage = 0; break;
```

Comment:
look like here tab after ``:``. in other lines was used spaces then better use spaces here too